test/service/test_response.py

```diff
@@ -2552,11 +2636,15 @@ def test_matrix_cell_count(self):
                         s['fileType']: s['matrixCellCount']
                         for s in hit['fileTypeSummaries']
                     }
-                    self.assertEqual(expected_counts, actual_counts)
+                    self.assertEqual(expected, actual_counts)
 
-        # Verify matrix_cell_counts across all hits in the 'files' endpoint
+        # Verify matrix cell counts across all hits in the 'files' endpoint
+        expected = {
+            'Rds': 54140 * 3,  # 3 analysis files
+            'fastq.gz': 0  # 24 sequence files
+        }
         actual_counts = Counter()
-        url = self.base_url.set(path='/index/files', args=params)
```

Comment:
I prefer this because it is more concise and no less general (as it would be for file system paths where we can't assume a path delimiter). The tuple form is only needed when interpolating dynamic values into a path because it ensure that those values are properly encoded.

```python
>>> furl('http://foo', path=('a', 'b/c'))
furl('http://foo/a/b%2Fc')
```